hub/powertoys/run.md

```diff
@@ -266,11 +256,11 @@ With the Window Walker plugin you can kill the process of a window if it hangs.
 
 #### File Explorer setting
 
-If the File Explorer settings in Windows are not set to open each window in a seperate process, you will receive the following information when searching for open Explorer windows:
+If the File Explorer settings in Windows are not set to open each window in a seperate process, you will receive the following message when searching for open Explorer windows:
```

Comment:
why do you change this? information is more clear here that it isn't a warning or error

Reply:
Because "the message" sounds more like something you actually **can click on**. It's an interactive element.

> After selecting the message, the "Folder options" window will open.